judge/models/tests/test_profile.py

```diff
@@ -0,0 +1,125 @@
+import base64
+import hmac
+import struct
+
+from django.conf import settings
+from django.test import TestCase
+from django.utils import timezone
+from django.utils.encoding import force_bytes
+
+from judge.models import Profile
+from judge.models.tests.util import CommonDataMixin, create_contest, create_contest_participation
+
+
+class OrganizationTestCase(CommonDataMixin, TestCase):
+    @classmethod
+    def setUpTestData(self):
+        super().setUpTestData()
+        self.profile = self.profiles['normal']
+        self.profile.organizations.add(self.organizations['open'])
+
+    def test_contains(self):
+        self.assertIn(self.profile, self.organizations['open'])
+        self.assertIn(self.profile.id, self.organizations['open'])
+
+        self.assertNotIn(self.profiles['superuser'], self.organizations['open'])
+        self.assertNotIn(self.profiles['superuser'].id, self.organizations['open'])
+
+        with self.assertRaisesRegex(TypeError, 'Organization membership test'):
+            'aaaa' in self.organizations['open']
+
+
+class ProfileTestCase(CommonDataMixin, TestCase):
```

Comment:
From codecov analysis, maybe check `__str__` since you do it in other models. This also applies to `SubmissionSource`.

Reply:
Done